packages/design-tokens/docs/ColorTokens.stories.tsx

```diff
@@ -0,0 +1,261 @@
+import React from "react"
+import { Box, Heading } from "@kaizen/component-library"
+import { Meta, Story } from "@storybook/react"
+import flatMap from "lodash.flatmap"
+import { useTheme } from "../react"
+import colorString from "color-string"
+
+export default {
+  title: "Design Tokens/Color Tokens",
+  parameters: {
+    layout: "fullscreen",
+    docs: {
+      description: {
+        component:
+          "This boarding is a representation of the design tokens available",
+      },
+    },
+  },
+} as Meta
+
+const Padding = ({
+  size = 1,
+}: {
+  size?: React.ComponentProps<typeof Box>["p"]
+}) => <Box p={size}> </Box>
+
+/**
+ * Use this for showing a simple horizontal or vertical stack of elements, with the support of padding/gaps between each of them.
+ */
+const Stack = React.memo(
+  ({
+    children,
+    gapSize,
+    childStyle,
+    horizontal,
+    ...divProps
+  }: {
+    children: React.ReactNode
+    childStyle?: React.CSSProperties
+    horizontal?: boolean
+    gapSize?: React.ComponentProps<typeof Padding>["size"]
+  } & JSX.IntrinsicElements["div"]) => {
+    const childrenArray = React.Children.toArray(children)
+    return (
+      <div
+        {...divProps}
+        style={{
+          display: "flex",
+          flexDirection: horizontal ? "row" : "column",
+          ...divProps.style,
+        }}
+      >
+        {flatMap(childrenArray, (child, i) =>
+          i === childrenArray.length - 1
+            ? [
+                <div style={{ flexShrink: 0, ...childStyle }} key={i}>
+                  {child}
+                </div>,
+              ]
+            : [
+                <div style={{ flexShrink: 0, ...childStyle }} key={i}>
+                  {child}
+                </div>,
+                <Padding key={`${i}-padding`} size={gapSize} />,
+              ]
+        )}
+      </div>
+    )
+  }
+)
+
+/**
+ * A component to show a simple color block with a name
+ */
+const ColorDemo = (props: { color: string; name?: string }) => {
+  const theme = useTheme()
+  const parsedColor = colorString.get(props.color)
+  return (
+    <div style={{ display: "flex", alignItems: "center" }}>
+      <h4
+        style={{
+          ...theme.typography.heading4,
+          writingMode: "vertical-lr",
+        }}
+      >
+        {props.name}
+      </h4>
+      <div
+        style={{
+          width: "20rem",
+          height: "10rem",
+          backgroundColor: props.color,
+          display: "flex",
+          flexDirection: "column",
+          justifyContent: "center",
+          alignItems: "center",
+        }}
+      >
+        {parsedColor && (
+          <Heading variant="heading-5">
+            <span
+              style={{
+                color: props.color,
+                filter: "invert(1) grayscale(1) contrast(100)",
+              }}
+            >
+              {colorString.to.hex(parsedColor.value)}
+            </span>
+          </Heading>
+        )}
+        {parsedColor && (
```

Comment:
This condition is the same as L100 so it's unnecessary (just merge them together with a fragment)